lapce-proxy/src/plugin.rs

```diff
@@ -143,10 +143,15 @@ impl PluginCatalog {
 
         let output = Pipe::new();
         let input = Pipe::new();
+        let env = match plugin_desc.get_plugin_env() {
+            Ok(env) => env,
+            Err(err) => return Err(err),
+        };
```

Comment:
```suggestion
        let env = plugin_desc.get_plugin_env()?;
```

Assuming this compiles, of course :)